### PR TITLE
BL-2219 Make pdf use custom styles css

### PR DESCRIPTION
**Pull request by gmartin7**
* Strip "OriginalImages/" off the front of css file paths
   before processing
* Of the Tests files only EnhancedImageServerTests has
   'real' changes the rest are due to adding a parameter to
   PretendRequestInfo()

### CODE REVIEW

**Comment by StephenMcConnel** [comment]
One line of actual code change, plus 5 new tests and various adjustments/enhancements to tests...  :+1:

File: src/BloomTests/PretendRequestInfo.cs [hunk 1]
```diff
@@ -14,11 +14,13 @@ public class PretendRequestInfo : IRequestInfo
 		public long StatusCode;
 		public string StatusDescription;
 
-		public PretendRequestInfo(string url)
+		public PretendRequestInfo(string url, bool forPrinting)
```
Comment (JohnThomson) [inline src/BloomTests/PretendRequestInfo.cs]: You could avoid a lot of changes and some extra code by declaring this as "bool forPrinting = false"

**Comment by JohnThomson** [comment]
One minor comment. Toss-up whether it's worth another commit. I don't have a problem merging as-is; just let me know.

File: src/BloomTests/PretendRequestInfo.cs (hunk 1 above)
Comment (gmartin7) [inline src/BloomTests/PretendRequestInfo.cs]: Good point...

**Pull request merged**